include/sys/ringbuf.h

```diff
@@ -23,6 +23,8 @@ static inline bool ringbuf_full(ringbuf_t *buf) {
 }
 
 void ringbuf_init(ringbuf_t *rb, void *buf, size_t size);
+void ringbuf_produce(ringbuf_t *buf, size_t bytes);
```

Comment:
Please document the functions appropriately. Give a hint that these are used when external entity (wrt. main processor) reads or write into the buffer and it must be allocated in physical memory.

Reply:
Ok, thanks - done